easycv/datasets/classification/data_sources/image_list.py

```diff
@@ -85,12 +85,20 @@ def parse_list_file(list_file, root, delimeter):
 
         fns = []
         labels = []
+        str_dict = dict()
 
         for l in lines:
             splits = l.strip().split(delimeter)
             fns.append(os.path.join(root, splits[0]))
-            # must be int,other with mmcv collect will crash
-            label = [int(i) for i in splits[1:]]
+            label = []
+            for i in splits[1:]:
+                # if i is digit, label = int(i); otherwise, label = str_dict[i]
+                if i.isdigit():
+                    label.append(int(i))
+                else:
+                    if i not in str_dict:
+                        str_dict[i] = len(str_dict)
+                    label.append(str_dict[i])
```

Comment:
The class id should be in the same order as the class name to ensure the consistance of inference